ui/src/components/Select.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+import CreatableSelect from 'react-select/creatable'
+import { KeyOption, keyOptions } from 'components/SelectData'
+import { ActionMeta, OnChangeValue } from 'react-select'
+
+export default class Select extends Component<{}> {

```

Comment:
would be nice to stay away from default exports if possible. Would also be nice to refactor this to be functional instead of a class component too